src/Setup/NativeSdkMsi/Headers.wxs

```diff
@@ -189,6 +189,9 @@
                 <File Source="sceutil.h" />
             </Component>
             <Component>
+                <File Source="sczutil.h" />
+            </Component>
+            <Component>
```

Comment:
Before I forget again, please don't @ me in a commit message.  I got way more notifications, like when you pushed the feature branch to GitHub.